org.eclim.jdt/java/org/eclim/plugin/jdt/command/complete/CodeCompleteCommand.java

```diff
@@ -185,6 +184,9 @@ protected CodeCompleteResult createCompletionResult(
     // of whether the user ever views it.
     /*return new CodeCompleteResult(
         kind, completion, menu, proposal.getAdditionalProposalInfo());*/
-    return new CodeCompleteResult(completion, menu, menu, type, offset);
+    CodeCompleteResult result = new CodeCompleteResult(completion, menu, menu, type, offset);
+    result.setRelevance(proposal.getRelevance());
```

Comment:
Sorting the completions by relevance should be an option (new command argument) since this will have an adverse affect on vim's code completion mechanism which, unfortunately, works best with completions that are sorted.